src/runtime/virtcontainers/clh.go

```diff
@@ -31,6 +31,8 @@ import (
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/types"
```

Comment:
Change looks good to me @fidencio. Just a nitpick about the commit message here. Replace
/Instead of passing to cloud-hypervisor the name of the bridge/Instead of passing to cloud-hypervisor the name of the tuntap device/

Since we were initially passing the name of the tuntap device before this change.